.ci/teamcity/oss/jest.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source "$(dirname "${0}")/../util.sh"
+
+export JOB=kibana-oss-jest
```

Comment:
Doesn't this run all the jest tests?

Reply:
Nope, just OSS - the rest are here: https://github.com/elastic/kibana/pull/85210/files#diff-3a783cd4dd85a375c09a88ca986ba1635156b73edd6405ed58ca4af5e8ff3857

Reply:
Oh, I figured the root jest script would default to the `jest.config.js` file which runs all the tests. My mistake

Reply:
The plan is to get there - but need to finish https://github.com/elastic/kibana/pull/84848 as the overhead of including all projects is a little too much at the moment.